pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -779,8 +780,9 @@ func makeFirewallName(name string) string {
 	return fmt.Sprintf("k8s-fw-%s", name)
 }
 
-func makeFirewallDescription(ipAddress string) string {
-	return fmt.Sprintf("KubernetesAutoGenerated_OnlyAllowTrafficForDestinationIP_%s", ipAddress)
+func makeFirewallDescription(serviceName, ipAddress string) string {
+	return fmt.Sprintf(`{"kubernetes.io/service-ip":"%s", "kubernetes.io/service-name":"%s"}`,
```

Comment:
@ArtfulCoder NB that this is a change for your firewall stuff - do you think it is OK?  Is this obvious enough?  We could add another label that says "is kube managed" but I don't think we need to.